iml-gui/crate/src/page/partial/header.rs

```diff
@@ -0,0 +1,219 @@
+                ],
+            ],
+            span![class![C.font_semibold, C.text_3xl, C.tracking_tight], "IML"],
+        ],
+        div![
+            class![C.block, C.lg__hidden,],
+            button![
+                class![
+                    C.flex,
+                    C.items_center,
+                    C.px_3,
+                    C.py_2,
+                    C.border,
+                    C.rounded,
+                    C.border_gray_400,
+                    C.text_gray_300,
+                    C.hover__text_white,
+                    C.hover__border_white,
+                ],
+                simple_ev(Ev::Click, Msg::ToggleMenu),
+                svg![
+                    class![C.fill_current, C.h_3, C.w_3],
+                    attrs! {
+                        At::ViewBox => "0 0 20 20",
+                    },
+                    title!["Menu",],
+                    path![
+                        attrs! { At::D => "M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z", },
+                    ],
+                ],
+            ],
+        ],
+        if model.menu_visibility == Visible {
+            div![
+                class![
+                    C.w_full,
+                    C.block,
+                    C.flex_grow,
+                    C.lg__flex,
+                    C.lg__items_center,
+                    C.lg__w_auto,
+                    C.lg__h_16,
+                ],
+                main_menu_items(&model),
+                div![a![
+                    attrs! {
+                        At::Class => "inline-block text-sm px-4 py-2 leading-none hover:text-teal-500 hover:bg-white mt-4 lg:mt-0",
```

Comment:
Could it be `class![...]`?

Reply:
Yes. This is a bit of cruft from initial conversion of the quickstart. I'll fix.